client/signup/jetpack-connect/plans-landing.jsx

```diff
@@ -56,10 +50,10 @@ class PlansLanding extends Component {
 				<PlansGrid
 					basePlansPath={ basePlansPath }
 					calypsoStartedConnection={ true }
-					hideFreePlan={ hideFreePlan }
+					hideFreePlan={ false }
 					intervalType={ intervalType }
-					isLanding={ isLanding }
-					landingType={ landingType }
+					isLanding={ true }
+					landingType={ 'jetpack' }
```

Comment:
You might want to remove the `vaultpress` specific code in `PlansGrid` with that change.